### PR TITLE
move dash-sql-editor to monorepo

### PR DESCRIPTION
**Pull request by chriskfwoo**
Issue for app: #[issue number here]

# App pull request
- [x] This is a new app
- [ ] I am improving an existing app (redesigns/code "makeovers")

## About

- Playground deployment URL (new version): https://dash-playground.plotly.host/dash-sql-editor/

## Workflow

- [x] I have created a branch in the appropriate monorepo, and the
  elements necessary for successful deployment are in place.
- [ ] If the app is a redesigned and/or restyled version of an
  existing gallery app, I've summarized the changes requested in the
  appropriate Streambed issue and confirm that they have been applied.
- [ ] If the app is on the Dash Gallery portal, I have added a link to
  the GitHub repository for the source code in the portal description.
- [ ] If the app is a reimplementation of a Python gallery app for the
  DashR gallery, the app in this PR mimics, as closely as possible,
  the style and functionality of the existing app.=
- [ ] I have removed *all Google Analytics code* from the app's
  `assets/` folder.

## The pre-review review

I have addressed all of the following questions:

- [x] Does everything in my code serve some purpose? (I have removed
  any dead and/or irrelevant code.)
- [x] Does everything in my code have a clear purpose? (My code is
  readable and, where it isn't, it has been commented appropriately.)]
- [x] Am I reinventing the wheel? (I have used appropriate packages to
  lessen the volume of code that needs to be maintained.)

### CODE REVIEW

File: apps/dash-sql-editor/app.py [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+    [
+        Input("sql__execute__handler", "n_clicks_timestamp"),
+        Input("sql__reset__handler", "n_clicks_timestamp"),
+    ],
+    [State("sql__textarea__input", "value"), State("log_storage", "data")],
+)
+def execute_query(btn_execute, btn_reset, value, history):
+
+    # reset button clicked
+    if int(btn_execute) < int(btn_reset):
```
Comment (shammamah) [inline apps/dash-sql-editor/app.py]: Why do these need to be cast to ints?

File: apps/dash-sql-editor/components/Column.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        6: "six",
+        7: "seven",
+        8: "eight",
+        9: "nine",
+        10: "ten",
+        11: "eleven",
+        12: "twelve",
+    }
+    return html.Div(
+        children, className="{} columns".format(number_mapping[width]), **kwargs
```
Comment (shammamah) [inline apps/dash-sql-editor/components/Column.py]: Nice :)

File: apps/dash-sql-editor/requirements.txt [hunk 3]
```diff
@@ -0,0 +1,6 @@
+dash==1.0.0
+plotly
```
Comment (shammamah) [inline apps/dash-sql-editor/requirements.txt]: Could you lock these versions as well?

**Comment by josegonzalez** [comment]
As per https://github.com/plotly/streambed/issues/13854, I am closing this as the new deployment process is in place.

If you need any changes from this branch, please recreate your changes against master and create a new pull request. Let me know if you need help and I am happy to jump in.

**Pull request closed without merging**